### PR TITLE
feat: add <Expandable /> permalinks

### PR DESCRIPTION
**Pull request by a-hariti**
add a `permalink` option to the `<Expandable />` component to render headers as links and show them up on the TOC.

### CODE REVIEW

File: docs/contributing/pages/components.mdx [hunk 1]
```diff
@@ -37,6 +37,25 @@ An Alert component with no level setting will render as a Note component.
 
 See also the [Note component](#note).
 
+## Expandable
+
+Render an expandable section.
+
+<Expandable title="Thi is a title">
```
Comment (lizokm) [inline docs/contributing/pages/components.mdx:44]:
```suggestion
<Expandable title="This is a title">
```

File: docs/contributing/pages/components.mdx [hunk 2]
```diff
@@ -37,6 +37,25 @@ An Alert component with no level setting will render as a Note component.
...
+This is some content
+</Expandable>
+
+```markdown {tabTitle:Example}
+<Expandable title="Thi is a title">
```
Comment (lizokm) [inline docs/contributing/pages/components.mdx:49]:
```suggestion
<Expandable title="This is a title">
```

**Review by lizokm — APPROVED** [review]
Thanks for updating!

File: docs/contributing/pages/components.mdx [hunk 3]
```diff
@@ -46,7 +46,7 @@ This is some content
 </Expandable>
 
 ```markdown {tabTitle:Example}
-<Expandable title="Thi is a title">
+<Expandable title="This is a title">
```
Comment (a-hariti) [inline docs/contributing/pages/components.mdx:49]: Thanks @lizokm

**Comment by chargome** [comment]
@a-hariti Nice!

**Pull request merged**